src/seedu/addressbook/data/person/Address.java

```diff
@@ -28,6 +28,14 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         this.value = address;
     }
 
+    @Override
+    public String getPrintableString(){
+        StringBuilder sb = new StringBuilder();
+        sb.append(" Address: ");
```

Comment:
Does it make sense for `getPrintableString()` to return a string with a leading space? Delimiting strings should be done in `getPrintableString(Printable... printables)` instead.